pom.xml

```diff
@@ -42,6 +42,8 @@
     <properties>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
+        <jenkins.version>1.651.2</jenkins.version>
```

Comment:
You shouldn't need to bump this dependency, EnvironmentContributor has existed for a long time.